site/src/Librecores/SiteBundle/Controller/DefaultController.php

```diff
@@ -40,8 +43,10 @@ public function homeAction(Request $request)
     /**
      * XXX: add caching for static pages
      * see http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/cache.html
+     *
+     * @Route("/{page}", name="librecores_site_page", requirements={"page"=".+"})
```

Comment:
The comment ` # allow / inside path` was pretty helpful to understand why the `requirements` option is added. Please restore that. Can you also add spaces before/after the `=` to make it `requirements = {` etc.

Reply:
@imphil Thank you for suggesting! I have made the changes. Please review them.